kfdefs/base/monitoring/alertmanager.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
   name: odh-monitoring
   namespace: opf-monitoring
 spec:
-  externalUrl: "http://alert-manager-opf-monitoring.apps.zero.massopen.cloud/"
+  externalUrl: ""
```

Comment:
I know were waiting on figuring out what to do with kfdef/base but shouldn't we just change this value to where the alertmanager is currently deployed: `https://alertmanager-main-openshift-monitoring.apps.smaug.na.operate-first.cloud`